types/koa-cors/koa-cors-tests.ts

```diff
@@ -0,0 +1,7 @@
+import * as Koa from 'koa';
+import * as KoaCors from 'koa-cors';
```

Comment:
`KoaCors` is a function here, so it should be `import = require`, same as `Koa`

Reply:
You're right, I make the change ASAP

Reply:
Done